lib/widgets/screens/profile/profile.dart

```diff
@@ -26,52 +119,198 @@ class ProfileScreenScroll extends StatefulWidget {
+                ],
+                motivation:
+                    'Sit amet justo donec enim diam vulputate ut pharetra sit amet aliquam id diam maecenas ultricies. Mi eget mauris pharetra et ultrices neque ornare aenean euismod elementum nisi quis eleifend.', //TODO
+              ),
+            ),
+          ],
+          if (widget.userData.offersHelp) ...[
+            const Divider(),
+            HowCanIHelpSection(
+              expertises: widget.userData.groupMemberships
+                  .firstWhere((g) => g.groupIdent == GroupIdent.mentors)
+                  .maybeWhen(
+                    mentorsGroupMembership: (g) =>
+                        g.expertises.map((e) => e.translatedValue!).toList(),
+                    orElse: () => [],
+                  ),
+              industries: widget.userData.groupMemberships
+                  .firstWhere((g) => g.groupIdent == GroupIdent.mentors)
+                  .maybeWhen(
+                    mentorsGroupMembership: (g) =>
+                        g.industries.map((e) => e.translatedValue!).toList(),
+                    orElse: () => [],
+                  ),
+              mentoringPreferences: const [
+                "Weekly check-ins",
+                "Monthly check-ins",
+                "Informal chats",
+                "Formal meetings",
+                "Spot mentoring",
+              ], //TODO
+              expectations:
+                  'Sit amet justo donec enim diam vulputate ut pharetra sit amet aliquam id diam maecenas ultricies. Mi eget mauris pharetra et ultrices neque ornare aenean euismod elementum nisi quis eleifend.', //TODO
             ),
-            const ProfileAboutMe(),
-            const ExperienceAndEducation(),
-            const AboutMyBusiness(),
-            const HowCanIHelpSection(),
           ],
-        ),
+          const Divider(),
+          ExperienceAndEducation(
+            experience: [
+              ExperienceInput(
+                position: "Marketing director",
+                companyName: "SVK Group",
+                start: DateTime.utc(2021, 12),
+                location: "Cleveland, Ohio, USA",
+              ),
+              ExperienceInput(
+                position: "Senior Marketing Manager",
+                companyName: "SVK Group",
+                start: DateTime.utc(2019, 6),
+                end: DateTime.utc(2021, 11),
+                location: "Cleveland, Ohio, USA",
+              ),
+              ExperienceInput(
+                position: "Founder",
+                companyName: "MC Consulting",
+                start: DateTime.utc(2016, 3),
+                end: DateTime.utc(2019, 5),
+                location: "Cleveland, Ohio, USA",
+                companyUrl: "https://mcconsulting.com",
+              ),
+            ],
+            education: [
+              EducationInput(
```

Comment:
this is called `AcademicExperience`, also in #135

Reply:
Fields are:
`institutionName`
`startDate`
`endDate` (optional)
`degreeType` (optional) -- i.e. "title" as listed here
`fieldOfStudy` (optional) --- i.e. "major" as listed here